### PR TITLE
grammar fixes

### PR DESCRIPTION
**Pull request by clairemuller**

### CODE REVIEW

**Review by webvictim — APPROVED** [review]
👍

File: docs/4.2/enterprise/quickstart-enterprise.md [hunk 1]
```diff
@@ -47,7 +47,7 @@ To start using Teleport Enterprise, you will need to Download the binaries and t
 After downloading the binary tarball, run:
 
 ```bsh
-$ tar -xzf teleport-ent-v{{ teleport.version }}-linux-amd64-bin.tar.gz
+$ tar -xzf <tar.gz>
```
Comment (benarent) [inline docs/4.2/enterprise/quickstart-enterprise.md]: Was there a specific reason to change this? I recently updated this so the version number will propagate to the UI and it'll look like this when rendered. 

image

File: docs/4.2/architecture/teleport_auth.md [hunk 2]
```diff
@@ -193,16 +189,15 @@ There are three types of data stored by the auth server:
...
+events and submits them to the auth server. The events recorded include:
 
 * successful user logins
 * node IP addresses
 * session time
 * session IDs
 
-!!! warning "Compatibility Warning":
-
+!!! warning "Compatibility Warning"
```
Comment (benarent) [inline docs/4.2/architecture/teleport_auth.md]: Thanks for fixing these.

File: docs/4.2/enterprise/quickstart-enterprise.md (hunk 1 above)
Comment (benarent) [reply docs/4.2/enterprise/quickstart-enterprise.md]: Yes, please bump it to 4.2.3. It's set here, https://github.com/gravitational/teleport/blob/master/docs/4.2.yaml#L35 
Please also update the SHA for OSS to `1c14362c9ba10f28088c7228b357dc6a70072d3d4afaa5510c70a8734068684c`

Comment (clairemuller) [reply docs/4.2/enterprise/quickstart-enterprise.md]: Awesome, will do!

**Pull request merged**